contracts/root/RootChain.sol

```diff
@@ -94,7 +94,7 @@ contract RootChain is RootChainStorage, IRootChain {
     headerBlock.start = nextChildBlock;
     headerBlock.end = dataList[2].toUint();
     require(
-      headerBlock.end > headerBlock.start,
+      headerBlock.end >= headerBlock.start,
```

Comment:
@0xAshish Take note of this change. Submitting a single block should be possible right?